src/components/calcite-input/calcite-input.tsx

```diff
@@ -176,15 +176,17 @@ export class CalciteInput {
     | "week" = "text";
 
   /** input value */
-  @Prop({ mutable: true, reflect: true }) value?: string = "";
+  @Prop({ mutable: true, reflect: true }) value?: string = null;
```

Comment:
Does this prop need to be reflected?

I think it should be undefined by default. Null is an explicit value and the optional '?' does nothing here.

```
@Prop({ mutable: true }) value?: string;
```